iceoryx_hoofs/platform/win/include/iceoryx_hoofs/platform/platform_settings.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+#ifndef IOX_HOOFS_LINUX_PLATFORM_PLATFORM_SETTINGS_HPP
+#define IOX_HOOFS_LINUX_PLATFORM_PLATFORM_SETTINGS_HPP
+
+#include <cstdint>
+
+namespace iox
+{
+namespace posix
+{
+class NamedPipe;
+}
+} // namespace iox
+
+// windows has two possible path separators
+constexpr const char IOX_PATH_SEPARATORS[] = "\\/";
+constexpr uint64_t IOX_UDS_SOCKET_MAX_MESSAGE_SIZE = 4096;
+constexpr char IOX_UDS_SOCKET_PATH_PREFIX[] = "";
```

Comment:
Why `IOX_UDS_SOCKET_*` when named pipes are used on windows?

Reply:
Since the unix domain sockets are using those numbers in their header and qnx/linux/mac do have different settings here. So I added windows as a stub.
But I will reduce the message size to zero.